symbolic-examples/stats/UInt.DivMod.32.Auto

```diff
@@ -1,4 +1,4 @@
 UInt.DivMod.32.Auto
-Number of constraints: 1387
-Number of variables: 1735
-Number of range lookups: 766
\ No newline at end of file
+Number of constraints: 1292
+Number of variables: 1642
+Number of range lookups: 770
```

Comment:
Hmm, it's interesting that the number of ranged constraints slightly increased in comparison to the code before optimizations